src/datastore/drivers/pg/PGStore.ts

```diff
@@ -34,15 +35,36 @@ export class PGStore implements Process, SQLStore {
         }
     }
 
-    public async createTransaction(): Promise<PGTransaction> {
+    public async createTransaction(
+        options?: {

```

Comment:
Generally in the library we don't use the nested options model.

I think it would be cleaner to use a default parameter here (i.e):
```(isolationLevel: PGIsolationLevel = PGIsolationLevel.SERIALIZABLE) => ```